spec/requests/proxying_spec.rb

```diff
@@ -2,8 +2,17 @@
 
 RSpec.describe "Proxying requests", type: :request do
   let(:body) { "abc" }
-  let(:upstream_path) { "/foo" }
-  let(:upstream_uri) { ENV['GOVUK_UPSTREAM_URI'] }
+  let(:upstream_slug) { "/foo" }
+  let(:upstream_path) { upstream_slug }
+  let(:upstream_domain) { ENV['GOVUK_UPSTREAM_URI'] }
+  let(:upstream_uri) { upstream_domain + upstream_path }
```

Comment:
I've got a bit confused by this variety of upstream variables. Do we need so many of them? I'm confused that we've got `upstream_slug` which isn't a slug and `upstream_domain` which is referencing a env var that describes it as a URI 🤔

Reply:
Have removed this commit 👍 agreed in hindsight this is more complex than it should be.